src/valuetype_idtype.ts

```diff
@@ -105,11 +105,20 @@ function parseIDType(def: ITypeDefinition, data: any[], accessor: (row: any, val
   return [];
 }
 
+function getMarkup(current): string {
```

Comment:
and what is `this` for this function. It seems like is has a meaning if so, please declare the type, too using `getMarkup(this: TYPE, current: TYPE)`